package.json

```diff
@@ -113,8 +111,8 @@
     "mutation:integration": "stryker run stryker.integration.config.js",
     "mutation:unit": "stryker run stryker.unit.config.js",
     "test": "npm run test:unit && npm run test:integration && npm run test:e2e && npm run test:compat",
-    "test:compat": "mocha test/compat/**/*.test.cjs",
-    "test:compat-all": "nve 10.13.0,12.0.0,14.0.0,16.0.0,18.0.0,19.0.0,20.0.0 mocha test/compat/**/*.test.cjs",
+    "test:compat": "ava test/compat/**/*.test.js",
+    "test:compat-all": "nve 14.18.0,16.13.0,18.0.0,19.0.0,20.0.0 npm run test:compat --ignore-scripts",
```

Comment:
The [Contributing Guidelines section on integration testing](https://github.com/ericcornelissen/shescape/blob/726b15d4f8641319e710b863ff167bd87255dff9/CONTRIBUTING.md?plain=1#L306-L308) should be updated accordingly.

Reply:
This is resolved with 8dd937cd4a9b596ad3d09dbbc5a82b7d36f64e89